src/lib/tests/renderRelayTree.tsx

```diff
@@ -0,0 +1,105 @@
+import { LoadingTestID } from "../utils/renderWithLoadProgress"
+import "DevTools/renderUntil"
+import { mount, RenderUntilPredicate } from "enzyme"
+import React from "react"
+import { Variables } from "relay-runtime"
+import { MockRelayRenderer, MockRelayRendererProps } from "./MockRelayRenderer"
+
+/**
+ * A {@link ReactWrapper.prototype.renderUntil} callback implementation that
+ * passes when no more loading indicators exist in the tree. Use this when you
+ * need to use `renderUntil` directly, such as after making updates to a Relay
+ * tree.
+ */
+export const RelayFinishedLoading: RenderUntilPredicate<any, any, any> = tree =>
+  !tree.find(`[testID=${LoadingTestID}]`).length
```

Comment:
The implementation of `renderRelayTree` in reaction uses a [class name](https://github.com/artsy/reaction/blob/master/src/DevTools/renderRelayTree.tsx#L15) as a loading sentinel. RN does not have classes, and (I think?) we need something that is flushed to a view primitive. I opted for View's [testID](https://facebook.github.io/react-native/docs/view#testid) prop. It comes with a caveat: 

> This disables the 'layout-only view removal' optimization for this view!

Though I think this is safe to ignore as we only pass it during tests.